controllers/k8ssandra/schemas.go

```diff
@@ -66,6 +81,10 @@ func (r *K8ssandraClusterReconciler) checkSchemaAgreement(mgmtApi cassandra.Mana
 		return result.Continue()
 	}
 
+	if _, found := versions["UNREACHABLE"]; len(versions) == 2 && found {
```

Comment:
This could use a comment, I can't make out from the code the reason for this check (why the magic word unreachable, why len check etc)

Reply:
@adutra has a better version of this function in #331. I'll update the impl in my branch accordingly.